pkg/constants/constants.go

```diff
@@ -94,6 +94,7 @@ var (
 	DefaultPrometheusPath                       = "/metrics"
 	QueueProxyAggregatePrometheusMetricsPort    = 9088
 	DefaultPodPrometheusPort                    = "9091"
+	EnableModelCache                            = KServeAPIGroupName + "/enable-model-cache"
```

Comment:
Can we name it model-cache instead so we have the flexibility of turning this into a non-Boolean field

Reply:
For example we might want to explicitly bind to the model cache resource